bouncy/distribution.yaml

```
@@ -604,6 +604,50 @@ repositories:
       url: https://github.com/ros2/navigation.git
       version: bouncy
     status: developed
+  navigation2:
+    doc:
+      type: git
+      url: https://github.com/ros-planning/navigation2.git
+      version: 0.1.0
+    release:
+      packages:
+      - costmap_queue
+      - dwb_controller
+      - dwb_core
+      - dwb_critics
+      - dwb_msgs
+      - dwb_plugins
+      - nav2_amcl
+      - nav2_astar_planner
+      - nav2_bringup
+      - nav2_bt_navigator
+      - nav2_costmap_2d
+      - nav2_costmap_world_model
+      - nav2_dijkstra_planner
+      - nav2_dwb_controller
+      - nav2_dynamic_params
+      - nav2_map_server
+      - nav2_mission_executor
+      - nav2_msgs
+      - nav2_robot
+      - nav2_simple_navigator
+      - nav2_system_tests
+      - nav2_tasks
+      - nav2_util
+      - nav2_voxel_grid
+      - nav_2d_msgs
+      - nav_2d_utils
+      - navigation2
+      tags:
+        release: release/bouncy/{package}/{version}
+      url: https://github.com/SteveMacenski/navigation2-release.git
+      version: 0.1.0-0
+    source:
+      test_pull_requests: true
+      type: git
+      url: https://github.com/ros-planning/navigation2.git
+      version: 0.1.
```

Looks like a typo here.
```suggestion
      version: 0.1.0
```

---

Thank you so much for that. That would have taken me far too long to identify myself.